playbooks/tasks/portal-role-task-mongo-setup-replicaset-init.yml

```diff
@@ -42,6 +41,6 @@
   fail:
     msg: |
       MongoDB replicaset initialization was not successful:
-      
+
       Mongo shell result: {{ mongo_shell_result }}
-  when: "mongo_shell_result.transformed_output | default('none output') != '{ \"ok\" : 1 }'"
+  when: 'mongo_shell_result.transformed_output | default(''none output'') != ''{ "ok" : 1 }'''
```

Comment:
Is this going to work correctly?
I think that single quotes will make it a string (not an expression) and will always evaluate to the same bool value.
I'm not even sure now that double quotes worked correctly.
Can we test it with a simple task?

Reply:
sneaky sneaky, I thought I reverted this but my code editor must have reset it on save